ocaml/testsuite/tests/typing-layouts-err-msg/value.ml

```diff
@@ -2,7 +2,6 @@
    flags = "-extension layouts_alpha"
    * expect
 *)
-(* CR layouts v2.9: all error messages below here are not reviewed for quality *)
```

Comment:
On line 83, "it's used as an element in a first-class module" reads funny to me - I'm not sure what an element of a first class module is, but it sounds like something we'd say about an expression, not a type declaration.  Maybe just "because it is a type declaration in a first-class module"?

Reply:
fixed